### PR TITLE
Add missing @ConfigItem annotations

### PR DESCRIPTION
**Pull request by loicmathieu**

### CODE REVIEW

**Comment by loicmathieu** [comment]
Thanks @machi1990 for the catch ;)

I'll provide more tests when I create the guide

File: extensions/liquibase-mongodb/runtime/src/main/java/io/quarkus/liquibase/runtime/LiquibaseMongodbConfig.java [hunk 1]
```diff
@@ -16,53 +17,68 @@
     /**
      * The migrate at start flag
      */
+    @ConfigItem
     public boolean migrateAtStart = false;
 
     /**
      * The validate on update flag
      */
+    @ConfigItem
```
Comment (machi1990) [inline extensions/liquibase-mongodb/runtime/src/main/java/io/quarkus/liquibase/runtime/LiquibaseMongodbConfig.java:26]:
```suggestion
    @ConfigItem(defaultValue="true")
```

File: extensions/liquibase-mongodb/runtime/src/main/java/io/quarkus/liquibase/runtime/LiquibaseMongodbBuildTimeConfig.java [hunk 2]
```diff
@@ -12,5 +13,6 @@
     /**
      * The change log file
      */
+    @ConfigItem
```
Comment (machi1990) [inline extensions/liquibase-mongodb/runtime/src/main/java/io/quarkus/liquibase/runtime/LiquibaseMongodbBuildTimeConfig.java:16]:
```suggestion
    @ConfigItem(defaultValue="db/changeLog.xml")
```

**Comment by loicmathieu** [comment]
@machi1990 thanks for the review, I updated my PR with defaultValue and more suitable init values

**Review by machi1990 — APPROVED** [review]
lgtm

**Comment by machi1990** [comment]
> @machi1990 thanks for the review, I updated my PR with defaultValue and more suitable init values

Thanks for the followup @loicmathieu

**Pull request merged**